static/js/upload.js

```diff
@@ -307,6 +340,8 @@ export function setup_upload(config) {
         if (info.type === "error") {
             // The remaining errors are mostly frontend errors like file being too large
             // for upload.
+            // TODO: It would be nice to keep the other uploads going if one fails,
+            // and show both an error message and the upload bar.
```

Comment:
Hmm, yeah, please open an issue for fixing this -- seems worth keeping track of.

Reply:
Opened #24287